clients/drcachesim/drcachesim.dox.in

```diff
@@ -734,6 +734,15 @@ To invoke the miss analyzer, pass \p miss_analyzer to the \p -simulator_type
 parameter. To write the prefetching hints to a file use the \p -LL_miss_file
 parameter to specify the file's path and name.
 
+For example, to run the analyzer on the benchmark
+\p clients/drcachesim/tests/stride_benchmark.cpp and store the prefetching
```

Comment:
Our release package does not include stride_benchmark, and someone using it may have no idea what this path is relative to: probably they would think the top level of the package.  I see 3 options: 1) We could add it to the package (and update this to point at the package path); 2) Change this to point at the source file via github URL or sthg, though that seems odd?  The user would probably wonder why we don't include it in the package?; or 3) Go back to what you originally had (generic app name).

Reply:
I favor the third option. We can add the detailed instructions for this specific benchmark in some other doc. Mircea, what do you think?

Reply:
Changes have been made based on our offline discussion.